src/theme.js

```diff
@@ -12,6 +12,8 @@ const addAliases = (arr, aliases) =>
 
 export const breakpoints = [32, 40, 48, 64]
 
+export const breakpointsUnit = 'em'
```

Comment:
Per discussed in Design System WG meeting, the `breakpointsUnit` doesn't seem to need to be exported. Please consider revert?